python-manual/modules/ROOT/pages/migration.adoc

```diff
@@ -0,0 +1,333 @@
+= Upgrade & Migration
+
+This page contains the list of new features and breaking changes that happened in the driver from version 4.4 to 5.LTS.
+For a full list of changes, see the link:https://github.com/neo4j/neo4j-python-driver/wiki/5.x-changelog[Driver -> Changelog].
+
+The link:https://github.com/stefano-ottolenghi/neo4j-drivers-migration-assistent[Neo4j Drivers Migration Assistent] simplifies the upgrade process, by contextualizing the changelog to your codebase.
+It scans your codebase for usage of deprecations and removals, and brings them up to you.
+The tool doesn't automatically rewrite your code; it only points at where action is needed, providing in-context information on how each hit should be addressed.
+
+Any of the 5.x versions of the driver is compatible both with Neo4j server 4.4 and 5.x, so you can upgrade the driver before you upgrade the server.
+
+[TIP]
+When upgrading the Neo4j server to a newer version, the Cypher queries in your application may also need updating. +
+See link:https://neo4j.com/docs/cypher-manual/current/deprecations-additions-removals-compatibility/[Cypher -> Deprecations, additions, and compatibility].
+
+
+== Recommended steps
+
+Using the migration assistent, the recommended upgrade steps are:
+
+1. Run the migration assistent with its default configuration and address all the issues that it surfaces. The `path` argument supports globbing and can be repeated as many times as needed. +
++
+[source, bash]
+----
+python main.py -l python <path...>
+----
++
+2. Re-run the assistent with the regex parser and check if any more legit issues are raised. +
++
+[source, bash]
+----
+python main.py -l python --regex-parser <path...>
+----
++
+The regex parser is able to parse implicit function calls and other hard-to-parse expressions, but is likely surface more false positive (for more information, see link:https://github.com/stefano-ottolenghi/neo4j-drivers-migration-assistent?tab=readme-ov-file#accuracy[Neo4j Drivers Migration Assistent -> Accuracy]).
+You can add any spurious entry to the ignored list, so that the assistent won't bring them up in future runs.
+3. Although the assistent can detect most of the locations where action is needed, a few changelog entries can't be surfaced in this form, so you should still revise the xref:breaking-changes[breaking changes] section and audit your codebase for the changes marked with {cross-mark}.
+4. You are ready to upgrade the driver package to the new version. +
++
+[source, bash]
+----
+pip install -U neo4j
+----
+
+
+== New features
+
+[cols="1a,1a"]
+|===
+
+|
+.Support for Python 3.12
+[%collapsible]
+====
+The driver is compatible with any Python version starting from 3.6 up to 3.12.
+====
+|
+.`asyncio` support
+[%collapsible]
+====
+For using the driver's features in parallel via the link:https://docs.python.org/3/library/asyncio.html[asyncio] package.
```

Comment:
```suggestion
For using the driver's features asynchronously via the link:https://docs.python.org/3/library/asyncio.html[asyncio] package.
```
I first wanted to suggest "concurrently", because async in Python is *not* parallel (same in JS, but not in all languages). No threading will be involved, only one Task can run at any given time.
But I'm not sold on "concurrently" either, because you can have that in 4.4 as well using threads. So there's no real differentiation besides the "asyncio" that follows.